ui/app/mainui/activitycenter/views/ActivityNotificationMessage.qml

```diff
@@ -2,57 +2,111 @@ import QtQuick 2.14
 import QtQuick.Layouts 1.14
 
 import StatusQ.Core 0.1
+import StatusQ.Core.Utils 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Components 0.1
 
 import shared 1.0
-import utils 1.0
-import shared.views.chat 1.0
 
 ActivityNotificationBase {
     id: root
 
-    property var messageContextMenu
+    readonly property string timestampString: Qt.formatTime(new Date(notification.timestamp), "hh:mm")
+    readonly property string timestampTooltipString: Qt.formatTime(new Date(notification.timestamp), "dddd, MMMM d, yyyy hh:mm:ss t")
```

Comment:
This is wrong, it doesn't respect the locale/app settings

Reply:
Furthermore, you probably want to format date as well, so just use sth like:
```qml
new Date(notification.timestamp).toLocaleString()
```

Reply:
Timestamp presentation here should be the same as chat message presentation. And to say is it wrong or not we should follow requirements. I checked chat figma - it's seems to be outdated. So we need @status-im/design-team to figure out what should be the timestamp presentation for message and notification tooltips

Reply:
You can't compare that to Figma... of course it will only be valid for English

Reply:
ok